src/crypto/CryptoNight.cpp

```diff
@@ -24,6 +24,34 @@
 
 #include "crypto/CryptoNight.h"
 
+// VARIANT ALTERATIONS
+#define VARIANT1_CHECK() \
+    if (MONERO && version > 6 && size < 43) \
+    { \
+        return false; \
+    }
+
+#define VARIANT1_INIT(part) \
+    const uint64_t tweak1_2_##part =\
+        (MONERO && version > 6) ? \
+            (*reinterpret_cast<const uint64_t*>(reinterpret_cast<const uint8_t*>(input) + 35 + part * size) ^ \
+            *(reinterpret_cast<const uint64_t*>(ctx->state##part) + 24)) : 0;
+
+#define VARIANT1_1(p) \
+    do if(version > 6) \
```

Comment:
Shouldn't this be `MONERO && version > 6` like other places?

Reply:
Current code there https://github.com/xmrig/xmrig/blob/dev/src/crypto/CryptoNight_monero.h